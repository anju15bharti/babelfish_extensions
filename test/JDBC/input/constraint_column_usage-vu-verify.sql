Use db_constraint_column_usage;
go

SELECT * FROM information_schema.CONSTRAINT_COLUMN_USAGE WHERE TABLE_NAME NOT LIKE 'sys%' ORDER BY COLUMN_NAME;
go

Use master;
go

SELECT * FROM information_schema.CONSTRAINT_COLUMN_USAGE WHERE TABLE_NAME NOT LIKE 'sys%' ORDER BY COLUMN_NAME;
go

use db_constraint_column_usage;
go

drop table tbl2;
go
drop table tbl1;
go
drop table tbl5;
go
drop table tbl3;
go
drop table sc1.tbl4;
go
drop schema sc1;
go

use master
go

drop database db_constraint_column_usage;
go
