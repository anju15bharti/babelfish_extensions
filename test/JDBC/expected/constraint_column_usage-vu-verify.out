Use db_constraint_column_usage;
go

SELECT * FROM information_schema.CONSTRAINT_COLUMN_USAGE WHERE TABLE_NAME NOT LIKE 'sys%' ORDER BY COLUMN_NAME;
go
~~START~~
nvarchar#!#nvarchar#!#nvarchar#!#nvarchar#!#nvarchar#!#nvarchar#!#nvarchar
db_constraint_column_usage#!#dbo#!#tbl1#!#arg1#!#db_constraint_column_usage#!#dbo#!#tbl1_pkey
db_constraint_column_usage#!#dbo#!#tbl5#!#arg10#!#db_constraint_column_usage#!#dbo#!#tbl5_arg10_arg11_fkey
db_constraint_column_usage#!#dbo#!#tbl5#!#arg11#!#db_constraint_column_usage#!#dbo#!#tbl5_arg10_arg11_fkey
db_constraint_column_usage#!#dbo#!#tbl2#!#arg3#!#db_constraint_column_usage#!#dbo#!#tbl2_pkey
db_constraint_column_usage#!#dbo#!#tbl2#!#arg4#!#db_constraint_column_usage#!#dbo#!#tbl2_arg4_fkey
db_constraint_column_usage#!#dbo#!#tbl3#!#arg5#!#db_constraint_column_usage#!#dbo#!#tbl3_pkey
db_constraint_column_usage#!#dbo#!#tbl3#!#arg6#!#db_constraint_column_usage#!#dbo#!#tbl3_pkey
db_constraint_column_usage#!#sc1#!#tbl4#!#arg7#!#db_constraint_column_usage#!#sc1#!#tbl4_check
db_constraint_column_usage#!#sc1#!#tbl4#!#arg8#!#db_constraint_column_usage#!#sc1#!#tbl4_check
~~END~~


Use master;
go

SELECT * FROM information_schema.CONSTRAINT_COLUMN_USAGE WHERE TABLE_NAME NOT LIKE 'sys%' ORDER BY COLUMN_NAME;
go
~~START~~
nvarchar#!#nvarchar#!#nvarchar#!#nvarchar#!#nvarchar#!#nvarchar#!#nvarchar
~~END~~


use db_constraint_column_usage;
go

drop table tbl2;
go
drop table tbl1;
go
drop table tbl5;
go
drop table tbl3;
go
drop table sc1.tbl4;
go
drop schema sc1;
go

use master
go

drop database db_constraint_column_usage;
go
